docs/content/documentation/getting-started/configuration.md

```diff
@@ -142,6 +142,10 @@ external_level = "error"
 paths = "on"
 taxonomies = "on"
 anchors = "on"
+# Whether to remove date prefixes for page path slugs.
+# For example, content/posts/2016-10-08_a-post-with-dates.md => posts/a-post-with-dates
+# When true, content/posts/2016-10-08_a-post-with-dates.md => posts/2016-10-08-a-post-with-dates
+paths_keep_dates = false
```

Comment:
Just bike shedding on the name: anyone has a better name? I'm not too sold on this one